src/blockchain/node/wallet/subchain/statemachine/Job.hpp

```diff
@@ -78,12 +78,15 @@ class Job : virtual public wallet::Job, public Actor<Job, SubchainJobs>
     ~Job() override;
```

Comment:
If we already touch this code, shouldn't this destructor be virtual if we inherit from this class?

Reply:
It already is, otherwise the override' qualifier would be complained about during compilation. I agree with Stroustrup that repeating the 'virtual' now that we can use 'override' or 'final' is bad style, which is why I am reluctant to change this.